frame/composable-traits/src/instrumental.rs

```diff
@@ -2,6 +2,28 @@ use codec::Codec;
 use frame_support::{pallet_prelude::*, sp_std::fmt::Debug};
 use sp_runtime::Perquintill;
 
+/// An indication of pool state. Shows whether the transfer of assets is currently taking place with the current pool.
+#[derive(Copy, Clone, Encode, Decode, Debug, PartialEq, MaxEncodedLen, TypeInfo)]
+pub enum State {
+	/// Indicates that there is currently no asset transfering going on for this asset
+	/// and it can be initialized.
+	Normal,
+	/// Indicates that an asset is currently being transferred from one pool to another
+	/// for this asset, so it is not possible to initialize a new transfer.
+	Transferrig,
+}
+
+/// An ndication of access rights for admin accounts.
+#[derive(Copy, Clone, Encode, Decode, Debug, PartialEq, MaxEncodedLen, TypeInfo)]
+pub enum AccessRights {
+	/// Account has full access rigths
+	Full,
+	/// Account has access only to `rebalance` function
+	Rebalance,
```

Comment:
We don't have an extrinsic of `rebalance` function, and so we should have this type of rights? If your answer is yes, then could you write a comment about this (TODO about implementing of an extrinsic of `rebalance` function or something else).

Reply:
Did it.